real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/baseNodes/SerialNode.java

```diff
@@ -28,11 +32,43 @@
 	int start = 0;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1804384399/issues/source?bid=2938665&fileBranchId=3059924#l33)